apps/www/.storybook/main.ts

```diff
@@ -0,0 +1,24 @@
+import { dirname, join } from "path"
+import type { StorybookConfig } from "@storybook/nextjs"
+
+const config: StorybookConfig = {
+  stories: [
+    "../registry/stories/**/*.mdx",
+    "../registry/stories/**/*.stories.@(js|jsx|ts|tsx)",
+  ],
+  addons: [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "@storybook/addon-styling",
```

Comment:
`@storybook/addon-styling` is deprecated in favor of `@storybook/addon-styling-webpack`

https://storybook.js.org/addons/@storybook/addon-styling
https://github.com/storybookjs/addon-styling-webpack

Reply:
Good catch, @stramel. In this case, because the `@storybook/nextjs` framework package is being used, we won't need either package to get tailwind working 😁

Reply:
thanks for the catch! I was upgrading to storybook v7.5.2 anyway so I took the opportunity to remove these 👍